packages/amplify-cli/src/index.ts

```diff
@@ -51,8 +52,30 @@ export async function run() {
       }
     }
     rewireDeprecatedCommands(input);
+
     const context = constructContext(pluginPlatform, input);
 
+    // Initialize feature flags
+    const contextEnvironmentProvider = new CLIContextEnvironmentProvider({
+      getEnvInfo: () => context.amplify.getEnvInfo(),
+    });
+
+    const getProjectPath = (): string => {
```

Comment:
this looks like the same as the above getProjectPath function. Should it be in a shared location?